manifests/base/argo-server/argo-server-certificate.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: argo-workflows-issuer
+spec:
+  selfSigned: {}
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: argo-server-cert
+spec:
+  dnsNames:
+  - argo-server.argo.svc.cluster.local
+  - argo-server.argo.svc
+  - argo-server
+  issuerRef:
+    kind: Issuer
+    name: argo-workflows-issuer
+  secretName: argo-server-tls
```

Comment:
Did you intend for this to be deployed as part of installing our manifests? I see manifests/base/argo-server/kustomization.yaml calls out the yaml files in this directory. Should we add it to that?

Reply:
Thanks, added to `manifests/base/argo-server/kustomization.yaml`.

Yes, intend to install as part manifests, since certificate from `Certificate` resource is persistent compared to one argo-server generates itself. This way user will only need to set up trust once.